Gemfile.lock

```diff
@@ -820,7 +818,7 @@ DEPENDENCIES
   yui-compressor
 
 RUBY VERSION
-   ruby 2.6.5p114
+   ruby 2.7.6p219
```

Comment:
@pleary same here. It deployed to staging, interestingly. I was sort of expecting it to barf b/c it didn't know about that Ruby.